pkgs/development/python-modules/pure-python-adb-homeassistant/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+buildPythonPackage rec {
+  pname = "pure-python-adb-homeassistant";
+  version = "0.1.6.dev0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fe6d90220a6880649f6d6df4e707ce5034676710ee6146145ef995f7b769a482";
+  };
+
+  meta = with lib; {
+    description = "Pure python implementation of the adb client";
+    homepage = https://github.com/Swind/pure-python-adb;
```

Comment:
same here

Reply:
my guess would be that it is the same person (JeffLlrion), however i cannot pin-point like before.

Reply:
Even though there seems no appropriate branch at https://github.com/JeffLIrion/pure-python-adb, I'd say let's set this as upstream.